Tests/runtest.py

```diff
@@ -66,9 +66,11 @@
                 print(" %s should be: %s" % (value, expected_value))
                 sys.exit(1)
 
+        """
```

Comment:
I wrote that code to check other output files. But python random sorting of dictionaries makes printouts irreproducible.